### PR TITLE
Slightly change resource grid name column header text + row content tooltip

### LINKED ISSUE #1050: Additional text should have something in the title
image

While hover over shows what the values in lighter font is (container id/ process id), we should have something in title too. Current display makes it confusing UI. 

Also since we are writing long text for container Id (whole guid), we should use Process Id rather than PID.

### PR DESCRIPTION
**Pull request by adamint**
Fixes #1050

Adds (Id) to the end of the column header and as we already use Container Id, changes pid to Process Id.

### CODE REVIEW

File: src/Aspire.Dashboard/Components/Pages/Resources.razor [hunk 1]
```diff
@@ -42,7 +42,7 @@
             <FluentDataGrid Items="@FilteredResources" ResizableColumns="true" GridTemplateColumns="1fr 2fr 1fr 2fr 3fr 2fr 1fr 1fr" >
                 <ChildContent>
                     <PropertyColumn Property="@(c => c.ResourceType)" Title="Type" Sortable="true" />
-                    <TemplateColumn Title="Name" Sortable="true" SortBy="@_nameSort">
+                    <TemplateColumn Title="Name (Id)" Sortable="true" SortBy="@_nameSort">
```
Comment (tlmii) [inline src/Aspire.Dashboard/Components/Pages/Resources.razor:45]:
```suggestion
                    <TemplateColumn Title="Name (ID)" Sortable="true" SortBy="@_nameSort">
```

We've been inconsistent about this in the past (and so I'm guessing there's a few places to clean up) but I think the current intent is to make sure to use `ID` when it is displayed to the user.

File: src/Aspire.Dashboard/Components/ResourcesGridColumns/ResourceNameDisplay.razor [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <span><FluentHighlighter HighlightedText="@FilterText" Text="@Resource.Name" /></span>
     @if (Resource is ProjectViewModel projectViewModel)
     {
-        var title = $"PID: {projectViewModel.ProcessId}";
+        var title = $"Process Id: {projectViewModel.ProcessId}";
```
Comment (tlmii) [inline src/Aspire.Dashboard/Components/ResourcesGridColumns/ResourceNameDisplay.razor:7]:
```suggestion
        var title = $"Process ID: {projectViewModel.ProcessId}";
```

File: src/Aspire.Dashboard/Components/ResourcesGridColumns/ResourceNameDisplay.razor [hunk 3]
```diff
@@ -19,7 +19,7 @@
     }
     else if (Resource is ExecutableViewModel executableViewModel)
     {
-        var title = $"PID: {executableViewModel.ProcessId}";
+        var title = $"Process Id: {executableViewModel.ProcessId}";
```
Comment (tlmii) [inline src/Aspire.Dashboard/Components/ResourcesGridColumns/ResourceNameDisplay.razor:22]:
```suggestion
        var title = $"Process ID: {executableViewModel.ProcessId}";
```

**Review by tlmii — APPROVED** [review]

File: src/Aspire.Dashboard/Components/ResourcesGridColumns/ResourceNameDisplay.razor (hunk 3 above)
Comment (smitpatel) [reply src/Aspire.Dashboard/Components/ResourcesGridColumns/ResourceNameDisplay.razor:22]: but container uses Container Id in tool tip but ID in tool tip copy. Inconsistency will trigger some OCD at some point.

Comment (tlmii) [reply src/Aspire.Dashboard/Components/ResourcesGridColumns/ResourceNameDisplay.razor:22]: Hence my other comment above that we have a few places we need to clean up 😊

**Pull request merged**